libs/ballot-interpreter-nh/src/convert.ts

```diff
@@ -40,12 +41,21 @@ import {
 import { DefaultMarkThresholds } from './interpret';
 import { interpretBallotCardLayout } from './interpret/interpret_ballot_card_layout';
 import { Bit, FrontMarksMetadata, PartialTimingMarks, Size } from './types';
+import {
+  ParseConstitutionalQuestionError,
+  parseConstitutionalQuestions,
+} from './convert/parse_constitutional_questions';
 
 export { interpret } from './interpret';
 
+const debug = makeDebug('ballot-interpreter-nh:convert');
+
 function makeId(text: string): string {
   const hash = sha256(text);
-  return `${text.replace(/[^-_a-z\d+]+/gi, '-')}-${hash.substr(0, 8)}`;
+  return `${text.replace(/[^-_a-z\d+]+/gi, '-').slice(0, 64)}-${hash.slice(
```

Comment:
Truncating to 64 here was because some of the IDs for the constitutional questions were crazy long.